app/uk/gov/hmrc/eusubsidycompliance/models/EmailRequest.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.eusubsidycompliance.models
+
+import play.api.libs.json.{Json, OFormat}
+
+case class EmailRequest(
+  undertakingIdentifier: String,
+  businessEntityIdentifier: String,
+  messageType: String,
```

Comment:
is `messageType` a String in their schema? I think it should be an Integer

Reply:
Having checked the docs it seems to be a String! I thought it should of been an Int at first too, but Alas :D